packages/alt-ergo/alt-ergo.0.95.2/opam

```diff
@@ -1,24 +1,34 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
```

Comment:
because the configure file is versionned on the git (one should do autoconf to generate it from configure.in)